policy/modules/session/wayland.if

```diff
@@ -0,0 +1,79 @@
+## <summary>Policy for wayland desktops.</summary>
+
+#########################################
+## <summary>
+##	Associate the specified domain with the
+##	Wayland compositor attribute.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`wayland_compositor_domain',`
+	gen_require(`
+		attribute wayland_compositor;
+		attribute wayland_compositor_tmpfs_type;
+	')
+
+	typeattribute $1 wayland_compositor;
+	typeattribute $2 wayland_compositor_tmpfs_type;
+')
+
+#########################################
+## <summary>
+##	Connect and talk over the Wayland
+##	socket.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+## 	</summary>
+## </param>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+## 	</summary>
+## </param>
```

Comment:
This doesn't seem to fit `wayland_client_tmpfs_type`.

Reply:
Same question as above.

Reply:
Should be answered above.